hs_core/views/__init__.py

```diff
@@ -339,6 +339,31 @@ def delete_resource(request, shortkey, *args, **kwargs):
 
     return HttpResponseRedirect('/my-resources/')
 
+
+def rep_res_bag_to_irods_user_zone(request, shortkey, *args, **kwargs):
+    if request.is_ajax:
```

Comment:
@pkdash Have addressed all your review comments so far and pushed the changes. Let me know if you see any further issues.